packages/web/src/components/test-tracks-table/TestTracksTable.tsx

```diff
@@ -209,6 +210,11 @@ export const TestTracksTable = ({
     return moment(track.date).format('M/D/YY')
   }, [])
 
+  const renderAddedDateCell = useCallback((cellInfo) => {
```

Comment:
Same here potentially?

Reply:
yea, could definitely move render functions without deps outside of the component. Seems nice to have all of the render functions together, but ill move them if I add another 👍